src/Aggregation/Limit.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ConnectHolland\MongoAggregations\Aggregation;
+
+/**
+ * Represents an $limit aggregation stage in the pipeline.
+ *
+ * @author Jaap Romijn <jaap@connectholland.nl>
+ */
+class Limit extends AbstractAggregation
+{
+    /**
+     * Creates a new limit aggregation object.
+     *
+     * @param number $limit
+     */
+    public function __construct($limit = null)
+    {
+        if (!empty($limit)) {
```

Comment:
This should be ```isset``` because 0 would be an unusual but valid value but it wouldn't pass the test